CMake/build.cmake

```diff
@@ -13,23 +15,37 @@ if(NOT CMAKE_BUILD_TYPE)
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif()
 
+# SENSEI should always be position independent
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
We should consider adding this on the target directly.

```cmake
    set_target_properties(yourTargetName PROPERTIES
        POSITION_INDEPENDENT_CODE ON
        WINDOWS_EXPORT_ALL_SYMBOLS ON
    )
```

Reply:
we don't support windows, and have no plans to do so